tests/integration/targets/inventory_aws_mq/playbooks/tasks/mq_instance_create_cmd.yml

```diff
@@ -0,0 +1,26 @@
+---
+- include_tasks: find_broker.yml
+- block:
+  - name: Get engine versions
+    command: >
+      aws mq  describe-broker-engine-types --engine {{ engine }}
+    register: describe_engine_result
+
+  - name: Select latest engine version
+    set_fact:
+      engine_version: "{{ describe_engine_result.stdout | from_json | json_query('BrokerEngineTypes[0].EngineVersions') | map(attribute='Name') | sort | max }}"
+
+  - name: Create minimal MQ instance in default VPC and default subnet group
+    command: >
+      aws mq create-broker
```

Comment:
Why don't you use the Ansible mq_broker module to create the broker https://github.com/ansible-collections/community.aws/blob/main/plugins/modules/mq_broker.py?

Reply:
I did try that (commented out in the initial commit, and removed now), but I think it has a bug where the broker does not get tagged, therefore it fails the keyed group test for this plugin